docs/manual/developer/08_content_tests.md

```diff
@@ -84,3 +84,18 @@ This uses the build datastreams so the project must be rebuilt in order for chan
 The macro `stig_srg_mapping` in `tests/CMakeList.txt` should be used when adding a product for this test.
 
 This script uses `tests/stig_srg_mapping.py` to run the test.
+
+## Removed Rules
+This test ensures no rules are removed from the data stream.
+This test requires a built version of the "old" content to compare.
+In CI we use the last upstream release.
+Curenntly, this is only enabled for RHEL products.
+To this test on your own machine follow the example below.
+You should replace `0.1.76` with the latest release of the project.
+
+1. `wget https://github.com/ComplianceAsCode/content/releases/download/v0.1.76/scap-security-guide-0.1.76.zip`
+1. `unzip scap-security-guide-0.1.76.zip`
+1. `export ADDITIONAL_CMAKE_OPTIONS="-DENABLE_CHECK_RULE_REMOVAL:BOOL=ON -DOLD_RELEASE_DIR=scap-security-guide-0.1.76"`
```

Comment:
I wasn't able to run it the test if the options were set this way. It always failed with a traceback No such file or directory. I had to provide a full absolute path in `-DOLD_RELEASE_DIR`. With the absolute path it worked fine.